src/Tokenomia/CLI.hs

```diff
@@ -239,25 +243,26 @@ data Action
 
 instance DisplayMenuItem Action where
   displayMenuItem item = case item of
-    WalletList            -> " [Wallet]  - List Registered Wallets" 
-    WalletDisplay         -> " [Wallet]  - Display wallet utxos"
+    WalletList               -> " [Wallet]  - List Registered Wallets" 
+    WalletDisplay            -> " [Wallet]  - Display wallet utxos"
     WalletDisplayWihtinIndexRange   
-                          -> " [Wallet]  - Display wallet utxos within child address index range"     
-    WalletRestore         -> " [Wallet]  - Restore Wallets from your 24 words seed phrase (Shelley Wallet)"
-    WalletCreate          -> " [Wallet]  - Create a new Wallet"
+                             -> " [Wallet]  - Display wallet utxos within child address index range"     
+    WalletRestore            -> " [Wallet]  - Restore Wallets from your 24 words seed phrase (Shelley Wallet)"
+    WalletCreate             -> " [Wallet]  - Create a new Wallet"
     WalletGenerateChildAddresses 
-                          -> " [Wallet]  - Derive Child Adresses"
-    WalletCollateral      -> " [Wallet]  - Create a unique collateral for transfer"
-    WalletRemove          -> " [Wallet]  - Remove an existing Wallet"
-    TokenMint             -> " [Token]   - Mint with CLAP type policy (Fix Total Supply | one-time Minting and open Burning Policy)"
-    TokenBurn             ->  "[Token]   - Burn Tokens with CLAP type policy"
-    TokenTransfer         ->  "[Token]   - Transfer Tokens"
-    AdaTransfer           ->  "[Ada]     - Transfer ADAs"
-    VestingVestFunds      ->  "[Vesting] - Vest Funds"
-    VestingRetrieveFunds  ->  "[Vesting] - Retrieve Funds"
-    NodeStatus            ->  "[Node]    - Status"
-    NodeTranslateSlotToTime -> "[Node]    - Translate Slot To Time"
-    NodeTranslateTimeToSlot -> "[Node]    - Translate Time To Slot"
+                             -> " [Wallet]  - Derive Child Adresses"
+    WalletCollateral         -> " [Wallet]  - Create a unique collateral for transfer"
+    WalletRemove             -> " [Wallet]  - Remove an existing Wallet"
+    TokenMint                -> " [Token]   - Mint with CLAP type policy (Fix Total Supply | one-time Minting and open Burning Policy)"
+    TokenBurn                -> " [Token]   - Burn Tokens with CLAP type policy"
+    TokenTransfer            -> " [Token]   - Transfer Tokens"
+    AdaTransfer              -> " [Ada]     - Transfer ADAs"
+    VestingVestFunds         -> " [Vesting] - Vest Funds"
+    VestingRetrieveFunds     -> " [Vesting] - Retrieve Funds"
+    VestingVerifyPrivateSale -> " [Vesting] - Verify Private Sale"
+    NodeStatus               -> " [Node]    - Status"
+    NodeTranslateSlotToTime  -> " [Node]    - Translate Slot To Time"
+    NodeTranslateTimeToSlot  -> " [Node]    - Translate Time To Slot"
     ICOStatus                  ->  "[ICO]     - Status"
```

Comment:
Was this file fully formatted? Some arrows don't match up? Also, fourmolu should remove this indentation style, it may be set up wrong.

Reply:
I think this was for an existing module, so we wouldn't be applying fourmolu to it yet

Reply:
There has been an addition to it, but if we want to play by their rules for modules they started, we can discard the formatting

Reply:
I never ran fourmolu on this, but made some small changes with spacing when I started the project without thinking. Will undo the changes